<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Magic Radio Controller</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Teszt beállítások</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>LED fényerő:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="ledPwmSlider">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximum">
                 <number>255</number>
                </property>
                <property name="value">
                 <number>127</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="ledTurnerButton">
                <property name="text">
                 <string>LED kikapcsolása</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="flat">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Rádió adó</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="1" column="1">
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Hangfrekvencia</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QSlider" name="horizontalSlider">
                 <property name="maximum">
                  <number>2000</number>
                 </property>
                 <property name="value">
                  <number>440</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLabel" name="label_7">
                 <property name="font">
                  <font>
                   <pointsize>11</pointsize>
                   <weight>75</weight>
                   <bold>true</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>100 MHz</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QSpinBox" name="freqBox">
                   <property name="maximum">
                    <number>2000</number>
                   </property>
                   <property name="value">
                    <number>440</number>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Hz</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Adófrekvencia:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QPushButton" name="txButton">
                 <property name="text">
                  <string>Adás</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>67</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>RTTTL lejátszó</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>RTTTL kód:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="rtttlBox"/>
          </item>
          <item>
           <widget class="QProgressBar" name="playProgress">
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Sebesség:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="speedSlider">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>20</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="defaultSliderButton">
            <property name="text">
             <string>Visszaállít</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="flashLed">
            <property name="text">
             <string>LED villogtatás</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="simulationBox">
            <property name="text">
             <string>Szimulálás</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="playButton">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Lejátszás</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>402</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Eszk_z">
    <property name="title">
     <string>&amp;Eszköz</string>
    </property>
    <addaction name="actionConnect"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_openRTTTL"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Ve_ll_t_sok">
    <property name="title">
     <string>&amp;Beállítások</string>
    </property>
    <addaction name="action_Sound"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Eszk_z"/>
   <addaction name="menu_Ve_ll_t_sok"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>&amp;Csatlakozás</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Kilépés</string>
   </property>
  </action>
  <action name="action_openRTTTL">
   <property name="text">
    <string>RTTTL &amp;megnyitása</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Sound">
   <property name="text">
    <string>&amp;Hang</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>freqBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>freqBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>freqBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>setFrequency(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>314</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ledPwmSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>dimLed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionConnect</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>connectDevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setFrequency(int)</slot>
  <slot>dimLed(int)</slot>
  <slot>connectDevice()</slot>
  <slot>disableLED()</slot>
 </slots>
</ui>
